Samples/Widgets/SingleWidget/cpp-win32-packaged/SampleWidgetProviderApp/WidgetProvider.cpp

```diff
@@ -0,0 +1,205 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Random thought - how much of this code can be moved into a shared vcxproj between the single & multiple widgets examples?

Maybe the sample should just be the "multiple" one, as "1" is a valid value for "greater than zero" ? :)

Reply:
yup, sounds good. I'll remove single widget sample.